doc/source/deployment/requirements.rst

```diff
@@ -10,11 +10,11 @@ requirements.
 Infrastructure
 --------------
 
-* The `Deployer` must run openSUSE Leap 15 or SUSE Linux Enterprise 15. See
-  :ref:`setupdeployer` for required deployment tools and packages.
+* The `Deployer` must run openSUSE Leap 15.0 (**Neither Leap 15 nor Leap 15.1
+  will work**). See :ref:`setupdeployer` for required deployment tools and packages.
```

Comment:
not sure about "neither ..." statement, it si correct, but sounds negative :-)

Reply:
changed in 35a8815